### DISCUSSION TITLE
🎉🎉Who uses UI Kitten / feedback? 🎉🎉

### DISCUSSION THREAD

**Discussion by artyorsh** [akveo/react-native-ui-kitten#1336] [Show and tell]
Hi All! 

We are really keen to collect your feedback, and to start with, we'd like to know who is using UI Kitten. We would really appreciate if you:

- Share your company name
- Share your logo (in which case it might be used in the readme / frontpage etc)
- What kind of project? How large? (code base size or team size)
- If your project is open source or public, please share a link!
- What do you like most about UI Kitten, what could be done better?

Thanks to MobX for Questionnaire idea: mobxjs/mobx#681

**Comment by flexbox** [comment]
Hey @artyorsh 

> Share your company name

One of my client
Retail Shake - https://www.retailshake.com/

> Share your logo (in which case it might be used in the readme / frontpage etc)

![image](https://user-images.githubusercontent.com/360936/65870397-759a3100-e37c-11e9-9d32-3199255630c4.png)

> What kind of project? How large? (code base size or team size)

2 People

> If your project is open source or public, please share a link!

Not this one
But I have a meetup side-project https://github.com/flexbox/inspiration-native

> What do you like most about UI Kitten, what could be done better?

The api's are dead simple.
For example the `<Text>` have nice props and everything have the same phylosophy.
As I said on my meetup talk
![image](https://user-images.githubusercontent.com/360936/65870551-c1e57100-e37c-11e9-8b3d-dba671a37d3b.png)
https://courses.davidl.fr/presentations/react-native-web/#/

> What could be done better?

Some integration with the cli for generating screens (Onboarding, SignIn)
like in https://github.com/infinitered/ignite
Because the Demo iOS app is nice but it's not crystal clear on how to use the templates files

**Comment by sertiano93** [comment]
> Share your company name

No Company name yet, the plan is to create a company near the release of my first project (iOS & Android Application)

>  Share your logo (in which case it might be used in the readme / frontpage etc)

No logo yet either. Will try to update this page when I have it.
>  What kind of project? How large? (code base size or team size)

Developing a encrypted end-to-end chat application for both iOS & Android through React Native & expo. Team size is 2 people. One (me) for the front-end and the other guy for the back-end. 

>  If your project is open source or public, please share a link!

No open source for the moment, but I could provide a link in the near future for testing and get some feedback if there is interest. 

>  What do you like most about UI Kitten, what could be done better?

I really like that UI Kitten is so clean and give a nice feeling. The colors and components feels really professional and not something you created in some hours. This is why I feel I can use it into my project.
As with everything, things could be better. I can't really think of something that I wished existed in the library for the moment since I got everything I need. But I will come back and update this text later on =)

**Comment by darr1s** [comment]
Would be nice if Eva Design System have a Figma compatible version. I am currently working to fix the symbols in Figma, the whole Sketch symbol structure don't seem to be very compatible with Figma. Please let me know if you are interest to fix this together, I can clone and invite you.

FYI Figma is offering free-tier, built-in CSS export, better pen tools, multi-editor online collaboration, also a more direct approach on component (symbol).

**Comment by artyorsh** [comment]
@bin4ryio this was [already requested](https://github.com/eva-design/eva/issues/22). But good things first, we're already working on this and going to release Figma support approx. in the end of the month. Thanks :)

**Comment by ben-wright** [comment]
> * Share your company name

[TrackMyLifts](https://trackmylifts.io)

> * Share your logo (in which case it might be used in the readme / frontpage etc)

![image](https://user-images.githubusercontent.com/8125689/67598970-cd9f3a00-f766-11e9-8a40-4c0b73dc6af8.png)

> * What kind of project? How large? (code base size or team size)

Small/Medium sized project. Just me working on it!

It is a weight lifting fitness tracker app.

> * What do you like most about UI Kitten, what could be done better?

The documentation of UI Kitten is so great! I'm quite new to React Native and everything has been a breeze.

I can't think of anything that could be done better at this stage! I'll be sure to stay active on the issues and hopefully make some pull requests to help with the project!

Great job guys thanks!

**Comment by AndyHubert** [comment]
> Share your company name

Educational Resources and Services

> Share your logo (in which case it might be used in the readme / frontpage etc)

![ERAS](https://resourcingeducation.com/static/toad.svg)

> What kind of project? How large? (code base size or team size)

Two developers. [eReader platform](https://toadreader.com)

> If your project is open source or public, please share a link!

https://github.com/educational-resources-and-services/toad-reader-apps

> What do you like most about UI Kitten, what could be done better?

I like the overall structure and the theming in particular. I would like to see some improvements in how it works with react native web.

**Comment by globalkonvict** [comment]
Hello, 

I recently started with react native and been doing some work with native component and eventually found out about few native component library including NativeBase, and UiKitten.

Although I liked UiKitten most of all as it has well designed and clean components, and been using it in an app i am working on currently. Its a moderately large app and would probably maintain it for coming years.

I use it in my personal projects and i am a Student for now so no company info i can provide, but you have my gratitude.

Thank You

**Comment by muhsin-k** [comment]
@artyorsh First of all thanks for this awesome library.

**Company Details**


[Chatwoot](https://www.chatwoot.com/)


![appLogo](https://user-images.githubusercontent.com/12408980/76158593-5f6b7a80-613d-11ea-82b2-3634b2284970.png)


Hey All, I am Muhsin. One of the contributors of [Chatwoot](https://www.chatwoot.com/).


About Chatwoot, It is a customer support tool for instant messaging channels that can help businesses provide exceptional customer support through their websites or social media channels.

This was a product we started building in 2017. It failed due to a couple of obvious reasons. We built on Vue.js and Rails. It was in an inactive mode for around 1 and a half years. Recently we thought of putting it out instead of letting the code to rust. Our idea is to make it something like Gitlab/Mattermost where people can host their own version and we will provide a hosted version for people who don't want to self-host.


After we open-sourced, we received contributions from 50 developers all around the world.


Recently we have started building mobile application for [Chatwoot](https://github.com/chatwoot/chatwoot-mobile-app).  We are using [UI kitten](https://akveo.github.io/react-native-ui-kitten/) heavily in this project. Most of the components are based on UI kitten and its working very well with all kinds of mobile devices.


If anyone interested, all types of contributions are welcome (https://github.com/chatwoot/chatwoot-mobile-app). If you have any doubts/queries, you can email us at hello@thoughtwoot.com.

Thanks

**Comment by MirzayevFarid** [comment]
- We’re using UI Kitten at [Radity](https://radity.com/)

- ![logo-midnight@3x](https://user-images.githubusercontent.com/29518613/91811146-173a9a00-ec40-11ea-9d20-981cfcf59877.png)

- https://radity.com
- Radity develops digital products and build web and mobile applications (20 Team members)

Thanks for such an amazing work!

**Comment by rrebase** [comment]
Hey 👋

- Used UI Kitten to build my first React Native app [DogNote](https://dognote.app/ )

![image](https://user-images.githubusercontent.com/23059874/98442519-c8acee80-210d-11eb-993c-b8500fbac8e9.png)

- UI kitten has been great - excellent docs with an easy to use design system. Most of the components work well out-of-the-box with a clean look. Has saved a lot of time. 
- One thing I would change is reimplement the components using the latest reanimated v2. Currently I've had to use alternative components in some cases to prevent the FPS dropping below 60fps on native devices.

🎉 Thanks for the awesome work on this library!

**Comment by tolunayakbulut** [comment]
Hi everyone, 👋

we are using at [Global CTO Forum](https://www.globalctoforum.org)

![global-cto-forum](https://user-images.githubusercontent.com/31761883/101293947-21fb6100-3825-11eb-80e1-a0543d025db1.png)

Global CTO Forum connects world's leading CTOs and technology stakeholders.The Global CTO Forum (GCF) is a global independent organisation for Technology leaders and executives including CTOs, CIOs and Architects around the globe to come together and help its members succeed as well as give back to the society.
